javascripts/downloads.js

```diff
@@ -40,7 +45,7 @@ function onLoad() {
             if (oStableVersion === null) {
                 oStableVersion = oEntry;
             }
```

Comment:
please put it in the same line
} else if () {

Reply:
Implemented findings by Michael, ok now?